discord/guild.py

```diff
@@ -3438,8 +3433,6 @@ async def query_members(
         Request members that belong to this guild whose username starts with
         the query given.
 
-        This is a websocket operation and can be slow.
```

Comment:
You can just remove the "can be slow" part.

Reply:
Done